packages/program-boilerplate/src/transaction.ts

```diff
@@ -7,6 +7,7 @@ import {
 
 import {ProgramTriggerBody} from './types/rpc';
 import {ProgramType, User} from './types/saasquatch';
+const ObjectID = require('bson-objectid');
```

Comment:
Eric and I looked into turning this `require` into an `import` but it seemed to always throw an error